Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -16,6 +16,7 @@
 #include <QSignalBlocker>
 #include <QUrl>
 
+#include <QDirIterator>
```

Comment:
This is a small thing, but this include should be in the previous group of includes (with all the other Qt includes).